README.md

```diff
@@ -1,91 +1,23 @@
-# Mantle
+<p align="center">
+	<picture>
+		<source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/ngrok-oss/mantle/refs/heads/readme-revamp/.github/mantle-dark.svg">
+		<source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/ngrok-oss/mantle/refs/heads/readme-revamp/.github/mantle-light.svg">
+		<img alt="Mantle" src="https://raw.githubusercontent.com/ngrok-oss/mantle/refs/heads/readme-revamp/.github/mantle-light.svg" width="176" height="34" style="max-width: 100%;">
+	</picture>
+</p>
+<h1 align="center">
+  Mantle Design System
+</h1>
 
 [Mantle](https://mantle.ngrok.com) is [ngrok](https://ngrok.com/)’s UI library and design system. It’s inspired by [shadcn/ui](https://ui.shadcn.com) and uses [Radix](https://www.radix-ui.com) with [Tailwind](https://tailwindcss.com/) for styling. It’s documented using [Remix](https://remix.run/docs).
 
-## Get Started
+## Getting Started
 
-To get started, install `@ngrok/mantle` with your preferred package manager:
+Check out our [install instructions](https://github.com/ngrok-oss/mantle/tree/main/packages/mantle#readme).
```

Comment:
Can't you do relative path references here?

Reply:
_Devs don't read_ https://github.com/ngrok-oss/mantle/pull/345#discussion_r1826020202 😏

Reply:
No, not that. The reference to the sub mantle package readme

Reply:
Oh wait it is I who cannot read, you're not talking about the image URLs. Yeah I'll fix this!

:homer-in-bushes-emoji: